Flaky test: org.keycloak.testsuite.forms.ResetPasswordTest#resetPasswordLink

## org.keycloak.testsuite.forms.ResetPasswordTest#resetPasswordLink
[Keycloak CI - Forms IT (firefox)](https://github.com/keycloak/keycloak/actions/runs/5450996174/jobs/9916987847)

```
java.lang.AssertionError: Expected LoginPage but was Keycloak Account Management (https://localhost:8543/auth/realms/test/account/?session_state=3287efae-ef5b-44cf-80d1-5a5a7990d659&code=f388e421-728a-46cb-8fbe-d79823c38166.3287efae-ef5b-44cf-80d1-5a5a7990d659.a0584b2d-5cf7-4044-87bc-c66a70655251)
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.keycloak.testsuite.pages.AbstractPage.assertCurrent(AbstractPage.java:47)
	at org.keycloak.testsuite.pages.LoginPage.open(LoginPage.java:236)
...
```

--- a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/ResetPasswordTest.java
+++ b/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/ResetPasswordTest.java
@@ -163,6 +163,7 @@ public class ResetPasswordTest extends AbstractTestRealmKeycloakTest {
         openResetPasswordUrlAndDoFlow(resetUri, "account", oauth.AUTH_SERVER_ROOT + "/realms/test/account/");
 
         AccountHelper.logout(testRealm(), username);
+        WaitUtils.waitForPageToLoad();
 
         TestAppHelper testAppHelper = new TestAppHelper(oauth, loginPage, appPage);
         testAppHelper.login(username, "resetPassword");

